src/pages/app/schema/ModelSchema.ts

```diff
@@ -30,7 +30,7 @@ export const getOrganisationSchema = (): Schema[] => [
     sortable: true,
     required: true,
     inputType: InputType.NUMBER,
-    validator: yup.number().min(14, 'Enter a valid CAC number')
+    validator: yup.number().min(14, 'Enter a valid CAC number').max(14,'enter a valid CAC number')
```

Comment:
You can use [this](https://github.com/jquense/yup#stringlengthlimit-number--ref-message-string--function-schema) instead